README.md

```diff
@@ -1,13 +1,113 @@
 # Node.js API for MaxMind minFraud Score, Insights, and Factors
 
+## Description
+This package provides an API for the [MaxMind minFraud Score, Insights, and
+Factors web services](https://dev.maxmind.com/minfraud/).
+
+## Requirements
+
+MaxMind has tested this API with Node.js versions 8 and 10.  We aim to support
+active LTS versions of Node.js, as well as the latest stable release.
+
+## Installation
+
+```
+yarn add @maxmind/minfraud-api-node
+```
+
+If you are not able to use `yarn`, you may also use `npm`:
+
+```
+npm install @maxmind/minfraud-api-node
+```
+
+## API Documentation
+
+Documentation for this API can be found [here](https://maxmind.github.io/minfraud-api-node/)
+
+## Usage
+To use this API, first create a new `Client` object. The constructor
+takes your MaxMind account ID and license key. For example:
+
+```
```

Comment:
Nice to have but not necessary, adding a language identifier to these code blocks.